### PR TITLE
Add script covering case when TTS.Speak gets UNSUPPORTED_RESOURCE and…

### PR DESCRIPTION
**Pull request by vvvakulenko**
… UI.PerformAudioPassThru gets WARNINGS from HMI
Script is a part of TS #666

### CODE REVIEW

**Comment by vvvakulenko** [comment]
@istoimenova Please review

File: test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+-- ToDo (vvvakulenko): remove after issue "ATF does not stop HB timers by closing session and connection" is resolved
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForUI_IsReady = require('user_modules/IsReady_Template/testCasesForUI_IsReady')
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua]: @vvvakulenko, I think that reference here should be for testCasesForTTS_IsReady, also:
user_modules/IsReady_Template/testCasesForTTS_IsReady

File: test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"},"PerformAudioPassThru")
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_initHMI')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua]: @vvvakulenko, please move the function after application is registered.

File: test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+function Test:Precondition_Check_audioPassThruIcon_Existence()
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua]: @vvvakulenko, this function is not needed in scope of the requirement. If you leave it, move it together with PutFile(see comment above) after application is registered.
If you remove it, don't forget to remove reference of testCasesForPerformAudioPassThru

File: test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua (hunk 1 above)
Comment (vvvakulenko) [inline test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua]: @istoimenova , updated in commit 5017f9b2f2dd3ed64b39ff7355b62513cf337fd4

File: test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua (hunk 2 above)
Comment (vvvakulenko) [inline test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua]: @istoimenova , updated in commit 5017f9b2f2dd3ed64b39ff7355b62513cf337fd4

File: test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua (hunk 3 above)
Comment (vvvakulenko) [inline test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua]: @istoimenova , updated in commit 5017f9b2f2dd3ed64b39ff7355b62513cf337fd4

**Comment by vvvakulenko** [comment]
@istoimenova , Please see updated script in commit 5017f9b2f2dd3ed64b39ff7355b62513cf337fd4

File: test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+-- ToDo (vvvakulenko): remove after issue "ATF does not stop HB timers by closing session and connection" is resolved
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForTTS_IsReady = require('user_modules/IsReady_Template/testCasesForTTS_IsReady')
+local testCasesForPerformAudioPassThru = require('user_modules/shared_testcases/testCasesForPerformAudioPassThru')
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua]: @vvvakulenko, testCasesForPerformAudioPassThru is not used, please remove the row.

**Pull request merged**